seq2science/rules/qc.smk

```diff
@@ -116,11 +116,11 @@ if config["trimmer"] == "trimgalore":
             """
 
 
-elif config["trimmer"] == "fastp":
+elif config["trimmer"] == "fastp" or "kmer_estimation" in config:
     ruleorder: fastp_qc_PE> fastp_qc_SE > fastp_PE > fastp_SE
 
 
-    rule fastp_qc_SE:
+    checkpoint fastp_qc_SE:
```

Comment:
I don't think this one needs to be a checkpoint. It isn't referenced anywhere at least.

Reply:
:+1: